rca/project_styleguide/templates/patterns/pages/development/development.html

```diff
@@ -0,0 +1,198 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static %}
+{% get_settings %}
+
+{% block body_class %}
+    app--development
+{% endblock %}
+
+{% block content %}
+
+    {% include "patterns/molecules/hero/hero.html" with modifier="medium" %}
+
+    <div class="page page--overlap">
+
+        <div class="page__notch grid">
+            <div class="page__notch-block bg bg--dark"></div>
+        </div>
+
+        <header class="page__header bg bg--dark">
+            <div class="title-area title-area--landing-page grid">
+                <div class="title-area__content">
+                    <nav class="title-area__breadcrumb" aria-label="Breadcrumb">
+                        {% include "patterns/molecules/breadcrumb/breadcrumb.html" %}
+                    </nav>
+                    {% if page.title %}
+                        <h1 class="title-area__heading heading heading--display-two">{{ page.title }}</h1>
+                    {% endif %}
+                </div>
+                {% if hero_image_credit or hero_image %}
+                    <div class="title-area__credit layout__@large-start-five layout__@large-span-one">
+                        {% include 'patterns/atoms/image-info/image-info.html' with credit=hero_image_credit item=hero_image %}
+                    </div>
+                {% endif %}
+            </div>
+            {% if page.introduction %}
+                <div class="introduction introduction--indented grid">
+                    <div class="introduction__container">
+                        <div class="section__heading heading heading--five">
+                            {{ page.introduction|richtext }}
+                        </div>
+                    </div>
+                </div>
+            {% endif %}
+
+            <section class="section bg bg--dark">
+
+                <div class="section__row section__row--last project-overview project-overview--condensed">
+                    <div class="section__content grid">
+                        <div class="project-overview__content layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
+                            <div class="project-overview__media">
+                                {% if page.introduction_image %}
+                                    {% include "patterns/organisms/image-video-block/image-video-block.html" with modal=page.video out_of_grid=True curriculum=False heading=False has_meta=False copy=False subheading=False image=page.introduction_image caption=page.video_caption video=page.video modifier="small-bottom-margin image-video-block--no-padding" %}
```

Comment:
shame there's no nice way in the template to pass a lot of vars through, just a lot of sideways scrolling : )

Reply:
We should refactor this component really, it's become far too convoluted, one for the backlog!